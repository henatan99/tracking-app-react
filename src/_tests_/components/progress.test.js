import React from 'react';
import { create } from 'react-test-renderer';

import Progress from '../../components/progress';

it('renders when no props passed', () => {
  const tree = create(<Progress />).toJSON;
  expect(tree).toMatchSnapshot();
});
