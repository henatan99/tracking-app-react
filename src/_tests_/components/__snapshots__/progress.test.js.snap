// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when no props passed 1`] = `[Function]`;
